docs/index.rst

```diff
@@ -14,12 +14,12 @@ The code is `available from GitHub <https://github.com/fairlearn/fairlearn>`_.
    
```

Comment:
The material towards the top also needs minor updating:
* title: `Welcome to Fairlearn!`
* the intro paragraph:
```
Fairlearn` is an open source project that seeks to ... unfairness.
The source code and documentation are hosted on `GitHub <https://github.com/fairlearn/fairlearn>`_. 
```

[the landing page will be fully replaced, so I don't think we should tweak it any more than this.]